.env.example

```
@@ -28,4 +28,12 @@ REDDIT_CLIENT_SECRET = <REDDIT_CLIENT_SECRET>
 
 CLOUDFLARE_API_TOKEN = <CLOUDFLARE_API_TOKEN>
 CLOUDFLARE_KV_NAMESPACE_ID = <CLOUDFLARE_KV_NAMESPACE_ID>
-CLOUDFLARE_ACCOUNT_ID = <CLOUDFLARE_ACCOUNT_ID>
\ No newline at end of file
+CLOUDFLARE_ACCOUNT_ID = <CLOUDFLARE_ACCOUNT_ID>
+
+# Sentri 
+SENTRI_LOGGER=true
+SENTRI_CI=false
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SENTRI_CI key should go before the SENTRI_LOGGER key

(UnorderedKey)